aesthetic_ants/penguin/enemy.py

```diff
@@ -0,0 +1,13 @@
+import pyglet
+import random as rand
+
+
+class Enemy(pyglet.sprite.Sprite):
+
+    def __init__(self, enemy_image):
+        pyglet.sprite.Sprite.__init__(self, enemy_image)
+        self.velocity_x, self.velocity_y = rand.randint(0, 100), rand.randint(0, 100)
```

Comment:
Likewise here, use `random.randint` instead of `rand.randint`
```suggestion
        self.velocity_x = rand.randint(0, 100)
        self.velocity_y = rand.randint(0, 100)
```